kvmagent/kvmagent/plugins/tensorfusion/service.py

```
@@ -0,0 +1,230 @@
+DEFAULT_LOG_LEVEL = ProcessExecutor.DEFAULT_LOG_LEVEL
+BYTES_PER_MB = ProcessExecutor.BYTES_PER_MB
+DEFAULT_SHMEM_SIZE = ProcessExecutor.SHM_SIZE
+
+
+class TensorFusionService(object):
+    """High-level facade for TensorFusion Worker lifecycle management."""
+
+    def __init__(self, event_notifier=None):
+        try:
+            self._gpu_details = NVIDIA.query_gpu_details()
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)